Hearthstone Deck Tracker/Windows/MainWindow/MainWindow_Export.cs

```diff
@@ -135,12 +135,33 @@ private void BtnExport_Click(object sender, RoutedEventArgs e)
 			var deck = DeckPickerList.SelectedDecks.FirstOrDefault();
 			if(deck == null)
 				return;
-			var fileName = Helper.ShowSaveFileDialog(Helper.RemoveInvalidFileNameChars(deck.Name), "xml");
-			if(fileName == null)
-				return;
-			XmlManager<Deck>.Save(fileName, deck.GetSelectedDeckVersion());
-			await this.ShowSavedFileMessage(fileName);
-			Logger.WriteLine("Saved " + deck.GetSelectedDeckVersion().GetDeckInfo() + " to file: " + fileName, "Export");
+			System.Windows.Forms.SaveFileDialog dialog = new System.Windows.Forms.SaveFileDialog();
```

Comment:
Adding using System.Windows.Forms caused two Clipboard calls to be ambiguous. So I accessed the SaveFileDialog like this.